curvefs/src/tools/list/curvefs_partition_list.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+ *  Copyright (c) 2021 NetEase Inc.
```

Comment:
2022, check all files created this time.

Reply:
> 2022, check all files created this time.

fix